pkg/api/types.go

```diff
@@ -1355,6 +1355,7 @@ const (
 	ClusterProfileAWSStackrox           ClusterProfile = "aws-stackrox"
 	ClusterProfileAWSSDCICD             ClusterProfile = "aws-sd-cicd"
 	ClusterProfileGCPSDCICD             ClusterProfile = "gcp-sd-cicd"
+	ClusterProfileAroRH                                = "aro-redhat-tenant"
```

Comment:
```suggestion
	ClusterProfileAroRH                   ClusterProfile = "aro-redhat-tenant"
```

Reply:
You'll have to format that properly as well

Reply:
oh, thx, done!